CHANGES.rst

```diff
@@ -266,6 +266,9 @@ astropy.config
 astropy.constants
 ^^^^^^^^^^^^^^^^^
 
+- Fixed inconsistent mass values when they are imported straight from
```

Comment:
If you don't think we need a change log, please let me know.

Reply:
I think we don't need a change log -- we are fixing the development version that has not yet been released.